CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.8.0 (2021-06-16)
+
+- Added support for the AG Grid options [`stopEditingWhenCellsLoseFocus`](https://www.ag-grid.com/angular-grid/grid-properties/#reference-editing) (AG Grid 25.2.0+) and `stopEditingWhenGridLosesFocus` with `SkyCellType.Date` columns. [#113](https://github.com/blackbaud/skyux-ag-grid/pull/113)
```

Comment:
Should we link to https://github.com/blackbaud/skyux-ag-grid/pull/111 as well as 113? I _think_ the support for `stopEditingWhenCellsLoseFocus` actually comes from that PR instead of 113. If so, does that distinction matter here?

Reply:
The options are essentially synonymous, and support for both options comes from making `SkyCellType.Date` columns use a popup cell editor, which was done in #113. The work in #111 unblocks `@skyux/ag-grid` from being installed with AG Grid 25.